### PR TITLE
Keep draft on non success

### PR DESCRIPTION
**Pull request by reedstrm**
If cnx-publishing responds with anything other than success, don't increment the local version number. This allows a simple retry of publishing, after correcting the problem.

depends on Connexions/cnx-archive#481

### CODE REVIEW

**Comment by reedstrm** [comment]
adding cnx-publishing#150 as a dependency. That *should* pull in  easybake

**Comment by reedstrm** [comment]
[BUG - Cannot republish content on beta.cnx.org](https://trello.com/c/N4QVj3RN/74-bug-cannot-republish-content-on-beta-cnx-org)

File: cnxauthoring/tests/intercept.py [hunk 1]
```diff
@@ -21,9 +21,8 @@
     )
 
 from cnxarchive import config
-from cnxarchive.database import initdb as archive_initdb
 from cnxarchive import main as archive_main
-from cnxpublishing.db import initdb as publishing_initdb
+from cnxdb.init import init_db
```
Comment (pumazi) [inline cnxauthoring/tests/intercept.py]: 🎊 YAY!🎉

**Comment by reedstrm** [comment]
[Change authoring's content acquisition/derivation logic to address cooked content API changes](https://trello.com/c/7pJU7aZr/118-change-authoring-s-content-acquisition-derivation-logic-to-address-cooked-content-api-changes)

**Pull request merged**